src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -3922,7 +3922,7 @@ INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','ActiveEffluxSpeci
 INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','ActiveEffluxSpecificWithTransporterInTarget_MM','TM * kcat * K_water_src * C_src / (Km + K_water_src * C_src)','Amount per time');
 INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','ActiveInfluxSpecificWithTransporterInSource_Hill','TM * kcat * (K_water_src * max(C_src;0))^alpha / (Km^alpha + (K_water_src * max(C_src;0))^alpha)','Amount per time');
 INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','ActiveInfluxSpecificWithTransporterInSource_MM','TM * kcat * K_water_src * C_src / (Km + K_water_src * C_src)','Amount per time');
-INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','BrainPgPToPlasma_Hill','Min(CP * f_int * V * kcat * (fu * max(C_pls;0))^alpha / ( Km^alpha + (fu * max(C_pls;0))^alpha) ; P_endothelial * SA * fu * C_pls) + CP * f_int * V * kcat * (K_water_int * max(C_int;0))^alpha / (Km^alpha + (K_water_int * max(C_int;0))^alpha)','Amount per time');
+INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','BrainPgPToPlasma_Hill','Min(TM * kcat * (fu * max(C_pls;0))^alpha / ( Km^alpha + (fu * max(C_pls;0))^alpha) ; P_endothelial * SA * fu * C_pls) + TM * kcat * (K_water_int * max(C_int;0))^alpha / (Km^alpha + (K_water_int * max(C_int;0))^alpha)','Amount per time');

```

Comment:
TM for Transporter Amount. This is what we use in another transports

Reply:
yep, looks good